### PR TITLE
Feature/add pipelines

### PR DESCRIPTION
**Pull request by MaxManthey**

### CODE REVIEW

File: .github/workflows/frontend-movie-recommender-build-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Frontend Pages Build and Deploy
+on: 
+  push:
+    branches: [ main ]
+    paths:
+      - "MovieRecommender/frontend/**"
```
Comment (andreas-eberle) [inline .github/workflows/frontend-movie-recommender-build-deploy.yml:6]: Also add the helm directory. If that is changed, we should also redeploy

File: .github/workflows/movie-recommender-build.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        uses: docker/build-push-action@v3
+        if: ${{ github.ref == 'main' }}
+        with:
+          context: "{{defaultContext}}:MovieRecommender/backend"
+          push: true
+          tags: ghcr.io/${{ github.repository_owner }}/${{ github.event.repository.name }}/movie-recommender:${{ github.sha }},ghcr.io/${{ github.repository_owner }}/${{ github.event.repository.name }}/movie-recommender:latest
+
+
+  deploy:
+    name: deploy to k8s
```
Comment (andreas-eberle) [inline .github/workflows/movie-recommender-build.yml:44]: where are the emojis?

File: .github/workflows/movie-recommender-build.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
+name: Movie Recommender Build
+
+on:
+  push:
+    branches: [ main ]
```
Comment (andreas-eberle) [inline .github/workflows/movie-recommender-build.yml:5]: I think we should run it for all branches and not limit it to main and PRs on main.

File: MovieRecommender/frontend/src/views/MovieRecommender.vue [hunk 4]
```diff
@@ -67,27 +67,22 @@ export default Vue.extend({
 </script>
 
 <style scoped>
-.recommender-wrapper {
-  display: flex;
-  flex-direction: column;
+#similar-movies {
```
Comment (andreas-eberle) [inline MovieRecommender/frontend/src/views/MovieRecommender.vue:70]: Are these changes intended here in this PR?

File: helm/MovieRecommender/backend/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: movie-recommender-bff
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
```
Comment (andreas-eberle) [inline helm/MovieRecommender/backend/Chart.yaml:5]: remove the generated comments. It's just boilerplate :D

File: helm/MovieRecommender/backend/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: movie-recommender-bff
+description: A Helm chart for Kubernetes
```
Comment (andreas-eberle) [inline helm/MovieRecommender/backend/Chart.yaml:3]: Maybe add a better description.

File: .github/workflows/frontend-movie-recommender-build-deploy.yml (hunk 1 above)
Comment (MaxManthey) [reply .github/workflows/frontend-movie-recommender-build-deploy.yml:6]: There is no helm directory for the frontend, just the backend because it's deployed on pages

File: .github/workflows/movie-recommender-build.yml (hunk 2 above)
Comment (MaxManthey) [reply .github/workflows/movie-recommender-build.yml:44]: 👍added✅  🔧😍emojis🤩🚀

File: MovieRecommender/frontend/src/views/MovieRecommender.vue (hunk 4 above)
Comment (MaxManthey) [reply MovieRecommender/frontend/src/views/MovieRecommender.vue:70]: I had to make some changes in the frontend to check if the action was triggered on a change in the folder but it does make more sense to group CSS Ids and classes together which is what I did here

File: helm/MovieRecommender/backend/Chart.yaml (hunk 6 above)
Comment (MaxManthey) [reply helm/MovieRecommender/backend/Chart.yaml:3]: Changed the description

File: .github/workflows/movie-recommender-build.yml (hunk 3 above)
Comment (MaxManthey) [reply .github/workflows/movie-recommender-build.yml]: Now runs on all branches

**Review by andreas-eberle — APPROVED** [review]
